dvc.yaml

```
@@ -0,0 +1,18 @@
+stages:
+  create_model:
+    cmd: python src/create_model.py
+    deps:
+    - data/train.tsv
+    - data/test.tsv
+    - data/validation.tsv
+    outs:
+    - output/model.joblib
+  get_data:
+    cmd: python src/data/download_files.py
```

Similar to above:
```suggestion
    cmd: pipenv run download_data
```
and then `download_data = python -m src.data.download_files`